fabfile.py

```diff
@@ -1251,6 +1251,22 @@ def check_if_database_exists():
         return not checkDatabase.failed
 
 
+def check_if_db_tables_exist():
+    with settings(warn_only=True):
+        checkDatabase = venvcmd('assembl-pypsql -1 -u {user} -p {password} -n {host} -d {database} "{command}"'.format(
+            command="SELECT count(*) from permission", database=env.db_database,
+            password=env.db_password, host=env.db_host, user=env.db_user))
+        return not checkDatabase.failed
+
+
+def check_if_first_user():
+    with settings(warn_only=True):
+        checkDatabase = venvcmd('assembl-pypsql -1 -u {user} -p {password} -n {host} -d {database} "{command}"'.format(
+            command="SELECT count(*) from public.user", database=env.db_database,
+            password=env.db_password, host=env.db_host, user=env.db_user))
+        return not checkDatabase.failed and int(checkDatabase.strip('()L,')) > 0
```

Comment:
This does not seem to check if we are the first user (like I would understand from function name), but if there are some users.